mailpoet/assets/js/src/public.tsx

```diff
@@ -48,7 +49,7 @@ jQuery(($) => {
     }
   }
 
-  $('.mailpoet_recaptcha').each((index, element) => {
+  $('.mailpoet_recaptcha').each((_, element) => {
```

Comment:
There is one more thing that I want to add that occurred to me after I submitted the review. Initially, I thought that `_` was the TS way to mark an unused parameter, but then I realized thanks to a conversation with @brezocordero that it is actually a prefix to mark an unused parameter (https://www.typescriptlang.org/docs/handbook/release-notes/overview.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals). With that in mind, I would suggest we use a more explicit name like `_unused`. This should help those that are not super familiar with TS to understand what is going on in this like. What do you think?

Reply:
Thanks for changing this detail, Sam. I would suggest we update the code in all the places where `_` was used and instead of just in this line.

Also, I would recommend avoiding doing changes in the commits after the PR was submitted for review like it was done for `_` -> `_unused`. In my opinion, adding new commits is preferred in those situations. This because it is harder for the reviewer to see what changed and what didn't change in the original commits when force push is used. What do you think?

Reply:
@rodrigoprimo thanks for the note, but I'm fine with both since even with an interactive rebase the reviewer is still able to see the older comment ( I should agree sometimes I get's harder to notice the improvement ), with this, said I'd agree with a separate commit after the PR has been submitted.

Reply:
> I would suggest we update the code in all the places where _ was used and instead of just in this line.

I think we still need to update the above.